android/moviesreloaded/src/main/java/com/walmartlabs/moviesreloaded/demo/bottomsheet/persistent/PersistentBottomSheetFragment.java

```diff
@@ -0,0 +1,57 @@
+package com.walmartlabs.moviesreloaded.demo.bottomsheet.persistent;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+
+import com.ern.api.impl.navigation.ElectrodeNavigationActivityListener;
+import com.ern.api.impl.navigation.ElectrodeNavigationFragmentConfig;
+import com.ern.api.impl.navigation.ElectrodeNavigationFragmentDelegate;
+import com.ern.api.impl.navigation.MiniAppNavigationFragment;
+import com.google.android.material.bottomsheet.BottomSheetBehavior;
+import com.walmartlabs.moviesreloaded.R;
+
+public class PersistentBottomSheetFragment extends MiniAppNavigationFragment {
+    private BottomSheetBehavior mBottomSheetBehavior;
+
+    @NonNull
+    @Override
+    protected ElectrodeNavigationFragmentDelegate createFragmentDelegate() {
+        ElectrodeNavigationFragmentConfig config = new ElectrodeNavigationFragmentConfig();
+        config.setFragmentLayoutId(R.layout.fragment_bottom_sheet_persistent);
+        config.setReactViewContainerId(R.id.react_view_container);
+        return new ElectrodeNavigationFragmentDelegate(this, config);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
```

Comment:
Please add the same `@NonNull`/`@Nullable` annotations as in the super class here.